File: aspnet-mvc-demo/AspnetMvcDemo/Controllers/Pages/UsersController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspnetMvcDemo.Controllers.Pages
{
    public class UsersController : Controller
    {

         public ActionResult Edit()
        {
            return View("~/Views/Pages/Users/Edit.cshtml");
        }

         public ActionResult List()
        {
            return View("~/Views/Pages/Users/List.cshtml");
        }

         public ActionResult Show()
        {
            return View("~/Views/Pages/Users/Show.cshtml");
        }

    }
}

File: laravel-demo/app/Http/Controllers/Pages/UsersController.php
<?php

namespace App\Http\Controllers\Pages;

use App\User;
use App\Http\Controllers\Controller;

class UsersController extends Controller
{
    public function edit()
    {
        return view('pages.users.edit', ['title' => 'Edit user - Pages']);
    }

    public function list()
    {
        return view('pages.users.list', ['title' => 'User list - Pages']);
    }

    public function view()
    {
        return view('pages.users.view', ['title' => 'View user - Pages']);
    }

}
